resource-managers/yarn/pom.xml

```diff
@@ -88,6 +88,13 @@
       <artifactId>hadoop-client</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
I'm not sure if we should use `jakarta.servlet-api` instead of  `javax.servlet-api` directly